app/controllers/step_by_step_pages_controller.rb

```diff
@@ -127,6 +141,11 @@ def publish_page(publish_intent)
     @step_by_step_page.mark_as_published
   end
 
+  def schedule_to_publish(schedule_intent)
+    StepNavPublisher.update(@step_by_step_page, schedule_intent)
```

Comment:
Is this line needed?

Reply:
Yes, it's needed - one of the tests fails otherwise (`status` isn't correctly reported).

Reply:
I don't think you need this line. When the scheduled date is saved, you don't need to re-save the draft you just need to tell publishing-api that a step-by-step is due to be published on a certain date, which is what `StepNavPublisher.schedule_for_publishing(@step_by_step_page)` does.

Reply:
Have now removed this line, but needed to call `step_by_step_page.mark_draft_updated` within the `schedule_for_publishing` method.

Reply:
Why?

Reply:
Ah, turns out `given_there_is_a_draft_step_by_step_page` didn't actually create a step by step which set the `draft_updated_at` field. Have fixed the test, which means no need to call `mark_draft_updated` anymore.